lib/osproject_android.rb

```diff
@@ -54,10 +54,10 @@ def add_sdk!
       directory_split = app_class_location.split('/', -1)
 
       application_name = directory_split[-1].split(".")[0]
-      com_index = directory_split.index "com"
+      
       range = directory_split.length - 2
-      package_directory = directory_split.slice(com_index..range)
-     
+      package_directory = directory_split.slice(range - 2..range)
```

Comment:
Is `range - 2..range` correct? I see just above we also subtract 2 from `directory_split.length`.
Also I don't understand if the value `2` is a static value we always want to index by. How did we come up with this value?